public/js/actions/breakpoints.js

```diff
@@ -22,15 +23,51 @@ function enableBreakpoint(location: Location) {
   return addBreakpoint(location);
 }
 
+function isOriginalLocation(state, location: Location) {
```

Comment:
Can you prefix this with an underscore? All top-level methods in an action file should be action creators, and we use that convention to indicate "internal" methods in here.